code/lib/store/template/stories/args.stories.ts

```diff
@@ -73,6 +73,7 @@ export const Events = {
     await within(canvasElement).findByText(/initial/);
 
     await channel.emit(UPDATE_STORY_ARGS, { storyId: id, updatedArgs: { test: 'updated' } });
+    await new Promise((resolve) => channel.once(STORY_ARGS_UPDATED, resolve));
```

Comment:
@tmeasday @ndelangen Can you look at this, I don't know much about this API. Does it makes sense to await this in every test for every renderer?

Reply:
we need to normalise our code as in many place you wait for UPDATED event to resolve and other no.
i took the oportunity to do that.